tools/syncer.md

```diff
@@ -73,25 +73,44 @@ batch = 10
 # The testing address for pprof. It can also be used by Prometheus to pull the syncer metrics.
 status-addr = ":10081"
 
-skip-sqls = ["ALTER USER", "CREATE USER"]
+# Notice: skip-sqls is abandoned, please use skip-ddls instead.
```

Comment:
Update this sentence to:

"Notice: skip-sqls is abandoned, and use skip-ddls instead."

Note: Do not use the word "please" in user documents, which is over-polite.